src/main/webapp/jsp/studentFeedbackQuestionSubmissionEdit.jsp

```diff
@@ -1,120 +1,52 @@
-                                            withStudentEmail(request.getParameter(Const.ParamsNames.STUDENT_EMAIL)).
-                                            withCourseId(request.getParameter(Const.ParamsNames.COURSE_ID)).
-                                            toString();
-
-                %>
-                    <div id="registerMessage" class="alert alert-info">
-                        <%= String.format(Const.StatusMessages.UNREGISTERED_STUDENT, data.student.name, joinUrl) %>
-                    </div>
-                <%
-                    }
-
-                    String submitAction = data.isModeration() ?
-                                            Const.ActionURIs.INSTRUCTOR_EDIT_STUDENT_FEEDBACK_SAVE :
-                                            Const.ActionURIs.STUDENT_FEEDBACK_QUESTION_SUBMISSION_EDIT_SAVE;
-                %>
-                <form method="post" name="form_student_submit_response" action="<%= submitAction %>">
-                    <jsp:include page="<%= Const.ViewURIs.FEEDBACK_SUBMISSION_EDIT %>" />
-
-                    <div class="bold align-center">
-                        <%
-                            if (data.isModeration()) {
-                        %>
-                                <input name="moderatedstudent" value="<%= data.getStudentToViewPageAs().email %>" type="hidden">
-                        <%
-                            }
-
-                            boolean isSubmittable = data.isSessionOpenForSubmission() || data.isModeration();
-                            if (data.bundle.questionResponseBundle.isEmpty()) {
-                        %>
-                                There are no questions for you to answer here!
-                        <%
-                            } else if (data.isPreview() || !isSubmittable) {
-                        %>
-                                <input disabled="disabled" type="submit" class="btn btn-primary" id="response_submit_button" data-toggle="tooltip" data-placement="top" title="<%= Const.Tooltips.FEEDBACK_SESSION_EDIT_SAVE %>" value="Submit Feedback" style="background: #66727A;">
-                        <%
-                            } else {
-                        %>
-                                <input type="submit" class="btn btn-primary" id="response_submit_button" data-toggle="tooltip" data-placement="top" title="<%= Const.Tooltips.FEEDBACK_SESSION_EDIT_SAVE %>" value="Submit Feedback">
-                        <%
-                            }
-                        %>
-                    </div>
-                    <br><br>
-                </form>
+            </c:when>
+            <c:otherwise>
+                <!-- Cannot be empty to prevent default navbar -->
+                &nbsp;
+            </c:otherwise>
+        </c:choose>
+    </c:set>
+</c:if>
+
+<c:set var="altFooter">
+    <!-- Cannot be empty to prevent default footer -->
```

Comment:
On second thought, a `<span></span>` will probably do better than a `&nbsp;` since it won't show up (and can't be highlighted). Would that work?

Also, The JSP comment tag should be better than the HTML comment tag here: `<%-- --%>`

(same goes for the header above)